inc/ticket.class.php

```diff
@@ -622,7 +622,6 @@ public static function item_add_user(Ticket_User $item, $type = CommonITILActor:
         $ticket->getFromDB($tickets_id);
         $groups_id = [];
 
-        self::removeAssignUsers($ticket, $users_id, $type);
```

Comment:
What is the value of `$users_id`? Does it properly include the two actors in your example?

Reply:
No, it takes care of users 1 by 1

Reply:
Simply removing the call to `removeAssignUsers()` might cause regressions elsewhere.
As suggested by @stonebuzz, please perform a `git bisect`, and tests would be welcome.

Reply:
apparently broken behaviour since version 2.9.8